myst_nbs/time_series/bayesian_var_model.myst.md

```diff
@@ -0,0 +1,769 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3.9.0 ('pymc_ar_ex')
+  language: python
+  name: python3
+---
+
+(Bayesian Vector Autoregressive Models)=
+# Bayesian Vector Autoregressive Models
+
+:::{post} November, 2022
+:tags: time series, vector autoregressive model, hierarchical
```

Comment:
We already have several notebooks using the hierarchical model tag so we should use the same for new ones: https://www.pymc.io/projects/examples/en/latest/blog/tag/hierarchical-model.html. Otherwise it will be listed as the only notebook tagged hierarchical (with the others tagged hierarchical model)

Reply:
Updated that tag now.